applications/dds_video/metadata.json

```diff
@@ -0,0 +1,36 @@
+{
+	"application": {
+		"name": "DDS Video",
+		"authors": [
+			{
+				"name": "Ian Stewart",
+				"affiliation": "NVIDIA"
+			}
+		],
+	"language": "C++",
+		"version": "1.0",
+		"changelog": {
+			"1.0": "Initial Release"
+		},
+		"holoscan_sdk": {
+			"minimum_required_version": "0.6.0",
+			"tested_versions": [
+				"0.6.0"
```

Comment:
Can you please confirm that the target Holoscan SDK is v0.6.0? Do you foresee any issues with updating this to v2.0.0 (latest)?